include/igl/triangle/triangulate.cpp

```diff
@@ -18,15 +18,17 @@
 #  define IGL_PREVIOUSLY_DEFINED_VOID VOID
 #  undef VOID
 #endif
-#define ANSI_DECLARATORS
+#define ANSI_DECLARATORS 1
```

Comment:
Do you know what this is used for?

I am curious because you made the change here, but this term is not used in any of the other changes you made.

Reply:
This is used inside Triangle itself to control the function definition style. It's technically fine as libigl has it (just a define with no value) but I prefer having a value set.